pkg/controller/status/status.go

```diff
@@ -79,6 +80,16 @@ type StatusManager interface {
 	SetMetaData(meta *metav1.ObjectMeta)
 }
 
+// This regex matches any characters that are not allowed in a Status Reason
+var reasonInvalidCharacters = regexp.MustCompile(`[^A-Za-z0-9_,:]`)
+
+// This regex matches any characters not allows at the start of a Status Reason
+var reasonInvalidStartCharacters = regexp.MustCompile(`^[^A-Za-z]*`)
```

Comment:
Why is this plural again for the start character and why does it have the asterisk?

Reply:
Never mind.

Reply:
I had to think about that a bit myself when I looked at this. I'll add more to the comments to try to clarify what it is used for.